cmake/DyninstOptions.cmake

```diff
@@ -0,0 +1,40 @@
+option(USE_OpenMP "Use OpenMP for parallel parsing" ON)
+
+option(
+  LIGHTWEIGHT_SYMTAB
+  "Use lightweight symtab interface for ParseAPI, ProcControl, and Stackwalker; disables DyninstAPI build"
+  OFF)
+
+option(SW_ANALYSIS_STEPPER "Use ParseAPI-based analysis stepper in Stackwalker" ON)
+
+option(BUILD_RTLIB_32 "Build 32-bit runtime library on mixed 32/64 systems" OFF)
+
+option(DYNINST_ENABLE_LTO "Enable Link-Time Optimization" OFF)
+
+option(ENABLE_DEBUGINFOD "Enable debuginfod support" OFF)
+
+option(STERILE_BUILD "DEPRECATED -- Do not use" OFF)
+mark_as_advanced(STERILE_BUILD)
+
+option(ADD_VALGRIND_ANNOTATIONS "Enable annotations for Valgrind analysis" OFF)
+
+option(ENABLE_STATIC_LIBS "Build static libraries as well?" OFF)
+
+option(DYNINST_DISABLE_DIAGNOSTIC_SUPPRESSIONS
+       "Disable all warning suppressions and frame size overrides." OFF)
+
+option(
```

Comment:
The value of this is a string, so it should use `set(... CACHE ...)` instead of `option`.

Reply:
Fixed.